sdk/tests/conformance2/textures/misc/tex-unpack-params-imagedata.html

```diff
@@ -40,24 +40,77 @@
 <script>
 var wtu = WebGLTestUtils;
 var gl = wtu.create3DContext(undefined, undefined, 2);
+var actual;
 
 description("TexImage3D from ImageData with unpack params");
 
 debug("TexImage3D from ImageData with UNPACK_IMAGE_HEIGHT set (crbug.com/804123)");
 
-let imageData = new ImageData(64, 64);
+// framebuffer for readback
+var fbo = gl.createFramebuffer();
+gl.bindFramebuffer(gl.FRAMEBUFFER, fbo);
+
+function makeTestData(size, start) {
+  let data = new Uint8ClampedArray(size);
+  for (let i = 0; i < size; ++i) {
+    data[i] = (start + i) % 256;
+  }
+  return data;
+}
+
+// source data
+const unpack_image_height = 7;
+const width = 4;
+const height = 4;
+const depth = 4;
+const srcWidth = width;
+const srcHeight = (height - 1) * unpack_image_height + height;
```

Comment:
Can you leave a comment about the shape of the thing you're trying to create? This equation is hard to verify.

Reply:
I think you want (depth-1)*unpack_height+height.

Reply:
I think you're right, thanks. I'll comment it.